tests/keys-test.js

```diff
@@ -11,7 +11,7 @@ describe('keys', function () {
     });
   });
 
-  describe('when there are many long keys', function () {
+  describe.skip('when there are many long keys', function () {
```

Comment:
@dustinsmith1024 - Did you intend to have this skipped?  I'm assuming this is part of the issue we were looking at the other day?